### PR TITLE
[sil_cameroon_azerty][sil_cameroon_qwerty] Major CSS Updates: Q-6.0.5, A-6.0.3

### PR DESCRIPTION
**Pull request by MattGyverLee**
This push is in response to discussions about the CSS not displaying properly on Android.
Version numbers, history, and dates updated.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/sil/sil_cameroon_azerty/source/sil_cameroon_azerty.kps [hunk 1]
```diff
@@ -221,7 +221,7 @@
         <Language ID="fub-Latn">Fulfulde, Adamawa</Language>
         <Language ID="fuv-Latn">Fulfulde, Mbororo</Language>
         <Language ID="gou-Latn">Gavar</Language>
-        <Language ID="gya-Latn">Gbaya, Northwest</Language>
+        <Language ID="gba-Latn">Gbaya, Northwest</Language>
```
Comment (darcywong00) [inline release/sil/sil_cameroon_azerty/source/sil_cameroon_azerty.kps:224]: The language tags will need to be updated in the `.keyboard_info` files for both keyboards too.

**Comment by darcywong00** [comment]
btw, the team won't be available to review keyboard PRs till next week. 🦃

File: release/sil/sil_cameroon_qwerty/source/sil_cameroon_qwerty.kmn [hunk 2]
```diff
@@ -3,7 +3,7 @@ store(&COPYRIGHT) '(c) 2018-2020 SIL Cameroon'
 store(&BITMAP) 'Cameroon.ico'
 store(&MESSAGE) 'Use ; or AltGr to access special characters.'
 store(&LAYOUTFILE) 'sil_cameroon_qwerty.keyman-touch-layout'
-store(&KEYBOARDVERSION) '6.0.4'
+store(&KEYBOARDVERSION) '6.0.5'
```
Comment (darcywong00) [inline release/sil/sil_cameroon_qwerty/source/sil_cameroon_qwerty.kmn:6]: Will you be addressing issue #1138 in a separate PR?

**Comment by MattGyverLee** [comment]
I'm making more changes because of https://github.com/keymanapp/keyboards/issues/1396 . You can expect more pushes.

**Comment by MattGyverLee** [comment]
I've tested iPad and iPhone (dark and light), android, and web. They look good to me. 
I'm just waiting for someone from WST to help me get Andika Africa Compact ( I sent a message to several people) to help with some OSK overflow issues with larger fonts.

**Comment by MattGyverLee** [comment]
Ok, I put in a local CSS fix for https://github.com/keymanapp/keyman/issues/4026 instead of replacing the font. I think this is ready for review.

**Comment by LornaSIL** [comment]
The QWERTY keyboard_info file still includes `gya-Latn`. It also includes `gba-Latn` so you should just delete the `gya-Latn` one.
I think everything else is okay.

**Comment by MattGyverLee** [comment]
> The QWERTY keyboard_info file still includes `gya-Latn`. It also includes `gba-Latn` so you should just delete the `gya-Latn` one.

Done, Thanks.
I hope one day Developer will help keep all this duplicate info in sync.

**Review by LornaSIL — APPROVED** [review]
LGTM

**Pull request merged**